utils/update-checkout

```diff
@@ -387,7 +402,8 @@ By default, updates your checkouts of Swift, SourceKit, LLDB, and SwiftPM.""")
     args = parser.parse_args()
 
     if args.reset_to_remote and not args.scheme:
-        print("update-checkout usage error: --reset-to-remote must specify --scheme=foo")
+        print("update-checkout usage error: --reset-to-remote must specify"
```

Comment:
Thanks, was too lazy to check more after your first comment, guess that's what happens!